content/authors/author/index.md

```diff
@@ -0,0 +1,8 @@
+

```

Comment:
Please edit your author bio in the proper Markdown format. Here's an [example](https://raw.githubusercontent.com/section-engineering-education/engineering-education/master/content/authors/adith-bharadwaj/index.md)